package_manifest.yaml

```diff
@@ -343,7 +345,7 @@ katello_packages:
     rubygem-hammer_cli_katello: {}
     rubygem-katello:
       releasers:
-        - koji-katello-jenkins
+        - "{{ nightly_releaser }}"
```

Comment:
@ehelms correct me if I'm wrong, but isn't this going to break releasing Katello? Even for actual Katello releases, it gets the source from the last successful Jenkins nightly build.

Reply:
It shouldn't. This change should be a noop but when https://github.com/theforeman/obal/pull/110 is merged we also need to set `nightly_package_tito_releaser_args`. I can update the PR so they're compatible.

Thinking about it, I'm wondering if we need a `release_is_nightly` parameter for jobs that are only nightly. Then after branching we can flip the boolean when we add the actual released sources.

Reply:
theforeman/obal#110 has been merged.